docs/source/workflow.rst

```diff
@@ -33,7 +33,7 @@ Adhoc tasks are often required to support one-time tasks, like a database
 migration.  Sometimes they are also useful for exploratory purposes, or even
 sometimes for end-to-end tests. PaaSTA supports this use case through ``paasta
 local-run``, which supports building images locally, as well as using the same
-image as a "live" deployment.
+image as a "live" Deployment.
```

Comment:
i think this is referring to the general concept of a deployment and not necessarily the K8s Deployment object, so this can probably be left alone